game-pathfinderwrathoftherighteous/index.js

```diff
@@ -1,46 +1,134 @@
+const Promise = require('bluebird');
 const path = require('path');
-const { fs, types, util } = require('vortex-api');
-const GAME_ID = 'pathfinderwrathoftherighteous';
-const NAME = 'Pathfinder: Wrath\tof the Righteous';
-const STEAM_ID = '1184370';
-const GOG_ID = '1207187357';
-function findGame() {
-    return util.GameStoreHelper.findByAppId([STEAM_ID, GOG_ID])
```

Comment:
The GameStoreHelper is a far more superior method to find the game as it queries all available game stores, not just Steam